esmvaltool/cmor/_fixes/CMIP5/inmcm4.py

```diff
@@ -48,3 +50,38 @@ def fix_data(self, cube):
         cube *= 0.01
         cube.metadata = metadata
         return cube
+
+
+class nbp(Fix):
+    """Fixes for nbp"""
+
+    def fix_file(self, filepath, output_dir):
+        """
+        Apply fixes to the files prior to creating the cube.
+
+        Should be used only to fix errors that prevent loading or can
+        not be fixed in the cube (i.e. those related with missing_value
+        and _FillValue or missing standard_name).
+
+        Parameters
+        ----------
+        filepath: basestring
+            file to fix.
+        output_dir: basestring
+            path to the folder to store the fix files, if required.
+
+        Returns
+        -------
+        basestring
+            Path to the corrected file. It can be different from the original
+            filepath if a fix has been applied, but if not it should be the
+            original filepath.
+
+        """
+        new_path = Fix.get_fixed_filepath(output_dir, filepath)
+        cube = iris.load_cube(filepath)
+        cube.standard_name = 'surface_net_downward_mass_flux_of_carbon_' \
```

Comment:
Use brackets instead of `\`:
```python
cube.standard_name = ('surface_net_downward_mass_flux_of_carbon_dioxide'
                      '_expressed_as_carbon_due_to_all_land_processes')
```